app/src/main/java/com/door43/translationstudio/ui/spannables/ArticleLinkSpan.java

```diff
@@ -15,9 +15,9 @@
  * Created by joel on 12/2/2015.
  */
 public class ArticleLinkSpan extends Span {
-    // e.g. [[en:ta:vol1:translate:translate_unknown|How to Translate Unknowns]]
-    // or [[:en:ta:vol1:translate:translate_unknown|How to Translate Unknowns]]
-    public static final Pattern ADDRESS_PATTERN = Pattern.compile("\\[\\[:?(([-a-zA-Z0-9]+:ta:[-\\_a-z0-9]+:[-\\_a-z0-9]+:[-\\_a-z0-9]+)(\\|(((?!\\]\\]).)+))?)\\]\\]");
+    // e.g. [[en:ta:vol1:translate:translate_unknown | How to Translate Unknowns]]
+    // or [[:en:ta:vol1:translate:translate_unknown | How to Translate Unknowns]]
+    public static final Pattern ADDRESS_PATTERN = Pattern.compile("\\[\\[:?(([-a-zA-Z0-9]+:ta:[-\\_a-z0-9]+:[-\\_a-z0-9]+:[-\\_a-z0-9]+)( ?\\|(((?!\\]\\]).)+))?)\\]\\]");
```

Comment:
Verified working!

![image](https://user-images.githubusercontent.com/34975907/120495510-c4d30900-c38a-11eb-9f0b-e430bf890ee4.png)

Reply:
I think the ` ?` should be replaced by a ` *`.  This is because ` ?` only matches up to one space, where ` *` will match up to any number of spaces.  Without the `*`, the following string will not match the regex:

`[[:en:ta:vol1:translate:translate_names  |  How to Translate Names]]`

It looks just like the regular string, but has **two** spaces around the pipe character.